src/vis/vis.js

```diff
@@ -319,6 +319,15 @@ var Vis = cdb.core.View.extend({
       this.https = true;
     }
 
+    // this checks that the screen is retina to add @2x to the url
+    if (window.devicePixelRatio > 1) {
+      data.layers.forEach(function(layer){
+        if(layer.type === "tiled"){
```

Comment:
I see this is only for ***basemaps***, but ... do all our supported basemaps follow that pattern?

We use that in our basemaps but I'm not sure if other providers follow that approach: Here, NASA and the like.